tensorflow/tools/dist_test/python/mnist_replica.py

```diff
@@ -52,33 +52,33 @@
 flags.DEFINE_boolean("download_only", False,
                      "Only perform downloading of data; Do not proceed to "
                      "session preparation, model definition or training")
-flags.DEFINE_integer("worker_index", 0,
-                     "Worker task index, should be >= 0. worker_index=0 is "
+flags.DEFINE_integer("task_index", None,
+                     "Worker task index, should be >= 0. task_index=0 is "
                      "the master worker task the performs the variable "
                      "initialization ")
-flags.DEFINE_integer("num_workers", None,
-                     "Total number of workers (must be >= 1)")
-flags.DEFINE_integer("num_parameter_servers", 2,
-                     "Total number of parameter servers (must be >= 1)")
+flags.DEFINE_integer("num_gpus", 1,
+                     "Total number of gpus for each machine."
+                     "If you don't use GPU, please set it to '0'")
 flags.DEFINE_integer("replicas_to_aggregate", None,
                      "Number of replicas to aggregate before parameter update"
                      "is applied (For sync_replicas mode only; default: "
                      "num_workers)")
-flags.DEFINE_integer("grpc_port", 2222,
-                     "TensorFlow GRPC port")
 flags.DEFINE_integer("hidden_units", 100,
                      "Number of units in the hidden layer of the NN")
 flags.DEFINE_integer("train_steps", 200,
                      "Number of (global) training steps to perform")
 flags.DEFINE_integer("batch_size", 100, "Training batch size")
 flags.DEFINE_float("learning_rate", 0.01, "Learning rate")
-flags.DEFINE_string("worker_grpc_url", None,
-                    "Worker GRPC URL (e.g., grpc://1.2.3.4:2222, or "
-                    "grpc://tf-worker0:2222)")
-flags.DEFINE_boolean("sync_replicas", False,
+flags.DEFINE_boolean("sync_replicas", True,
```

Comment:
Can we default this to False for now? There is an unresolved issue in sync_replicas training in open-source TensorFlow.